app/views/admin/adminLayout.scala.html

```diff
@@ -1,4 +1,4 @@
-@(tab: String)(content: Html)
+@(tab: String, config: play.api.libs.json.JsObject)(content: Html)
```

Comment:
I tried but it doesn't work. I think that it's because `Json.obj()` is not a literal representation of a value.